allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/NewRemoteWebDriver.java

```diff
@@ -297,6 +299,26 @@ public MenuElement elementMenu(By selector, By childsSelector, By subMenuSelecto
 		return driverExtention.elementMenu(selector, childsSelector, subMenuSelector, childsSubMenuSelector);
 	}
 	
+	@Override
+	public HorizontalSliderElement elementHorizontalSlider(final By selector) {
```

Comment:
Same here. Rename first parameter of constructors.

Reply:
Done.